modules/claims_api/app/sidekiq/claims_api/va_notify_declined_job.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VANotifyDeclinedJob < ClaimsApi::ServiceBase
+    LOG_TAG = 'va_notify_declined_job'
+
+    def perform(encrypted_ptcpnt_id, encrypted_first_name, representative_id)
+      lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+      ptcpnt_id = lockbox.decrypt(Base64.strict_decode64(encrypted_ptcpnt_id))
+      first_name = lockbox.decrypt(Base64.strict_decode64(encrypted_first_name))
+      representative = ::Veteran::Service::Representative.find_by(representative_id:)
+
+      if representative.blank?
+        raise ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
+          detail: "Could not find veteran representative with id: #{representative_id}"
+        )
+      end
+
+      res = send_declined_notification(ptcpnt_id:, first_name:, representative:)
+
+      ClaimsApi::VANotifyFollowUpJob.perform_async(res.id) if res.present?
+    rescue => e
+      msg = "VA Notify email notification failed to send with error #{e}"
+      slack_alert_on_failure('ClaimsApi::VANotifyDeclinedJob', msg)
+
+      ClaimsApi::Logger.log(LOG_TAG, detail: msg)
+
+      raise e
+    end
+
+    private
+
+    def find_poa(poa_code:)
+      ClaimsApi::PowerOfAttorney.find do |poa|
+        poa.form_data.dig('serviceOrganization', 'poaCode') == poa_code ||
+          poa.form_data.dig('representative', 'poaCode') == poa_code
+      end
+    end
+
+    def send_declined_notification(ptcpnt_id:, first_name:, representative:)
+      representative_type = representative.user_type
+      return send_organization_notification(ptcpnt_id:, first_name:) if representative_type == 'veteran_service_officer'
+
+      send_representative_notification(ptcpnt_id:, first_name:, representative_type:)
+    end
+
+    def send_organization_notification(ptcpnt_id:, first_name:)
+      content = {
+        recipient_identifier: ptcpnt_id,
+        personalisation: {
+          first_name: first_name || '',
+          form_type: 'Appointment of Veterans Service Organization as Claimantʼs Representative (VA Form 21-22)'
+        },
+        template_id: Settings.claims_api.vanotify.declined_service_organization_template_id
+      }
+
+      vanotify_service.send_email(content)
+    end
+
+    def send_representative_notification(ptcpnt_id:, first_name:, representative_type:)
+      representative_type_text = get_representative_type_text(representative_type:)
+
+      content = {
+        recipient_identifier: ptcpnt_id,
+        personalisation: {
+          first_name: first_name || '',
+          representative_type: representative_type_text,
+          representative_type_abbreviated: representative_type_text,
```

Comment:
Not sure if it is possible that these two variables could ever be empty, but if they are then the template will send back the `VANOTIFY400` error, so I would recommend adding the `|| ''` for lines 67 and 68 as well (did receive this from my local data for John Doe, again not sure if it can happen in Prod but might be better to be safe then sorry).

```suggestion
          representative_type: representative_type_text || '',
          representative_type_abbreviated: representative_type_text || '',
```